templates_cspm_eventbridge/OrgFullInstall.yaml

```diff
@@ -1,7 +1,33 @@
 AWSTemplateFormatVersion: "2010-09-09"
-Description: IAM Role and EventBridge resources used by Sysdig Secure       
```

Comment:
Can we leave this description as is? It was chosen by product

Reply:
Done. added correct description